### PR TITLE
feat: writing chapter about nginx

### PR DESCRIPTION
**Pull request by alemorvan**
#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

File: docs/books/web_services/022-web-servers-nginx.md [hunk 1]
```diff
@@ -1,10 +1,279 @@
...
+* Automatic index page generation;
+* Accelerated reverse proxy with cache;
+* Load balancing ;
+* Fault tolerance;
+* Cached support for FastCGI, uWSGI, SCGI and memcached cache server;
+* Various filters for gzip, xslt, ssi, image transformation, ...
+* Support for SSL/TLS and SNI;
+* HTTP/2 support.
+
+Other features :
```
Comment (SergeCroise) [inline docs/books/web_services/022-web-servers-nginx.md:56]:
```suggestion
Other features:
```

**Comment by sspencerwire** [comment]
@EzequielBruni Would you have time to give your technical opinion on the content here?  I know you've used Nginx some. @alemorvan requested this in [the documentation chat](https://chat.rockylinux.org/rocky-linux/pl/wagasgyzm3nyjm5gnfgmtyinsc).

**Comment by EzequielBruni** [comment]
I'll give it a look.

File: docs/books/web_services/022-web-servers-nginx.md [hunk 2]
```diff
@@ -1,10 +1,279 @@
...
+Nginx has a significant market share of 28% on the busiest sites on the market, just behind Apache (41%).
+
+#### Features
+
+Nginx offers the following basic functions:
+
+* Hosting of static web pages;
+* Automatic index page generation;
+* Accelerated reverse proxy with cache;
+* Load balancing ;
```
Comment (SergeCroise) [inline docs/books/web_services/022-web-servers-nginx.md:49]:
```suggestion
* Load balancing;
```

**Comment by alemorvan** [comment]
@sspencerwire please merge this document as this, as it is good enought for a first read, and later, as soon as ezequiel as time, we will open a new merge request. It just miss the Workshop part, the other part are ok.

Thx!

**Comment by sspencerwire** [comment]
> @sspencerwire please merge this document as this, as it is good enought for a first read, and later, as soon as ezequiel as time, we will open a new merge request. It just miss the Workshop part, the other part are ok.
> 
> Thx!

I should be able to get to this later today. I want to be able to merge and then edit it right away.

**Review by SergeCroise — APPROVED** [review]
LGTM!

**Review by sspencerwire — APPROVED** [review]

**Pull request merged**